example-server.js

```diff
@@ -562,6 +653,9 @@ async function handleVerifyVp(req, res) {
     if(error) {
       throw 'Invalid verifiable presentation id: ' + error;
     }
+    if(!vp.proof) {
```

Comment:
ok so I see what you're doing here: this is a mock server so I won't worry about this file to much as we're going to be passing the test vectors you wrote to various different endpoints and seeing what fails on actual implementations so good work here writing a mock server so quickly with all of these checks in it.